### DISCUSSION TITLE
Welcome to cs162_group_banking_project!

### DISCUSSION THREAD

**Discussion by Zoibderg** [Zoibderg/cs162_group_banking_project#1] [Announcements]
Hey guys! Here is the repo that we can use for the project. It may prove to be a lot easier to communicate and collaborate here as opposed to trying to do so in Canvas discussion boards.

**Comment by ghost** [comment]
Hey thanks. As I've said, I have no idea. I got some time this weekend and will try to watch some intro videos. I know its some sort of check-in/check-out? but that's about it

**Reply by Zoibderg** [reply]
It lets you store and manage code repositories, making it easy to track changes over time, work on projects with others, and keep backups. You can “commit” changes, “push” code updates, and create “branches” to work on new features without affecting the main codebase, then “merge” these changes when they’re ready. A lot of stuff can be done here in the browser, however I recommend looking into attaching remotes, pushing, and pulling! That's really the basics that you would need to operate outside the browser.
